lld/ELF/Arch/AArch64.cpp

```diff
@@ -999,7 +999,9 @@ class AArch64BtiPac final : public AArch64 {
 
 private:
   bool btiHeader; // bti instruction needed in PLT Header and Entry
-  bool pacEntry;  // autia1716 instruction needed in PLT Entry
+  bool pacEntry;  // Authenticated branch needed in PLT Entry
```

Comment:
maybe we can just have enum here instead of pair of bools? E.g.
``enum {
  Unsigned,
  SignedHint,  // use autia1716 instruction in PLT entry
  Signed // can use non-hint braa instruction
}
```

Reply:
Changed to an enum, thanks! It might be worth using a switch statement instead of if's and ternary operators at the end of `AArch64BtiPac::writePlt`, but it looks like that it's readable enough right now, so I left that "as is" unless there is a request for changing that as well.